release.md

```diff
@@ -112,3 +112,17 @@ Verify the latest version of `evalml` got installed by running
 ```shell
 python -c "import evalml; print(evalml.__version__)"
 ```
+
+# Miscellaneous
+## Add new maintainers to evalml-core-feedstock
+
+Per the instructions [here](https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list):
+1. Ask an existing maintainer to create an issue on the [repo](https://github.com/conda-forge/evalml-core-feedstock).
+  a. Select *Bot commands* and put the following title (change `username`):
```

Comment:
This isn't rendered on a new line in the doc:
![image](https://user-images.githubusercontent.com/4307001/182473088-18a45f55-12cc-4afe-9de2-3b855ec4998b.png)

Maybe fix or just combine everything into a single line. without the `a.` prefix.

Reply:
Fixed
https://github.com/alteryx/evalml/pull/3636/commits/33074fc4516677e982cc8d353c6072a6b12e091f